src/app/services/user-info-firebase.service.ts

```diff
@@ -42,11 +42,30 @@ export class UserInfoFirebaseService extends UserInfoService {
 
   override saveUser(user: UserData): Observable<string> {
     sessionStorage.setItem(user.uid,JSON.stringify(user));
-    return defer(() => from(setDoc(doc(collection(this.firestore, 'users'), user.uid), {
+    let userid = defer(() => from(setDoc(doc(collection(this.firestore, 'users'), user.uid), {
       uid: user.uid,
       ownedSets: user.getOwnedSetsToStore(),
       recentSets: user.getRecentSetsToStore()
     }) as Promise<void> ))
       .pipe(map((ret => user.uid as string)));
+
+      //Save user doesn't save to firebase unless this is here, it's 6:00am
+      userid.subscribe(a =>{
+        console.log("Save User")
+      })
+    return userid
+  }
+
+  override updateViewDate(studySet: StudySetData){
```

Comment:
Please add a space before the opening curly brace "{". 
```ts
override updateViewDate(studySet: StudySetData) {
```
Fix this everywhere in your code